src/foam/nanos/http/FileServlet.java

```diff
@@ -0,0 +1,63 @@
+package foam.nanos.http;
+
+import com.google.api.client.util.IOUtils;
+
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+public class FileServlet
+    extends HttpServlet
+{
+
+  public static final String SERVLET_NAME = "file";
```

Comment:
s/file/static/ means: replace "file" with "static"

It's a short-hand programmer notation.